### ISSUE TITLE
BSOD (win32kfull.sys) on dragging items - only when Volume² is running, in Windows 10 FCU

### ISSUE THREAD

**Issue by onedegreemediagroup** [irzyxa/Volume2#41]
This is a tough one, but I think there's enough evidence to suggest Volume² is involved.

After upgrading to the Windows 10 Fall Creator's Update, certain drag/drop operations (for some, dragging items on desktop or in file folders; for others, dragging selected text in Chrome, or dragging objects inside Adobe Illustrator) cause a Blue Screen of Death: SYSTEM_SERVICE_EXCEPTION, culprit driver Win32Kfull.sys.

In my case, this only occurs if Volume² is running. (In my case, v. 1.1.5.382). Running Volume² as Administrator or not appears to make no difference.

Other reports of this issue: 
https://answers.microsoft.com/en-us/windows/forum/windows_10-performance/systemserviceexception-win32kfullsys/a84467d3-b7a7-44c7-98ea-c6f14271f17c?messageId=29b3bafb-f36e-4fb7-9b1f-34753f62890b

https://www.reddit.com/r/Windows10/comments/7blljr/bsod_win32kfullsys_system_service_exception_when/

Any ideas on why Volume² would have any interaction with drag/drop events on the desktop or in other programs? 

Some reports indicate that downgrading to v. 1.1.4.347 seems to solve the issue - any idea what changed in Volume² after that point that would cause this?

I appreciate any attention you can devote to this issue - I use Volume² dozens, if not hundreds, of times per day, and would hate to lose it!

Thanks.

**Comment by onedegreemediagroup** [comment]
Glad it helped - I know the feeling 😢 

I'm not yet totally convinced the downgrade is enough to fix, but maybe the restart is the key.

I ran the 1.1.4.347 installer, and let it auto-uninstall the previous 1.1.5.382 version.  Thought I was in the clear, but got another BSOD after a couple of hours.  However, I didn't restart the system as you did.

Now that I've restarted (thanks to the BSOD), and am running 1.1.4.347 without any leftover files in memory from 1.1.5.382, I'll hope I'm good to go.  Will report back if I hit another one.
